training/training_code/README.txt

```diff
@@ -0,0 +1,4 @@
+1. place your training code into this directory 
```

Comment:
Can you please incorporate the instructions into `README.md` and remove this file and update any references to this file?